### PR TITLE
CMTA: Austin, Texas metro area public transit

### PR DESCRIPTION
**Pull request by nickturskyi**
Turns out, HAFAS is used not only by the European customers! :)

I'm not 100% confident in this code, as all this is discovered by exploratory testing of the official app & other profiles in `hafas-client`, but it works! Feedback is welcome of course.

### CODE REVIEW

File: p/cmta/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const _createParseLine = require('../../parse/line')
+const _parseLocation = require('../../parse/location')
+const _createParseJourney = require('../../parse/journey')
+const _createParseDeparture = require('../../parse/departure')
+const _formatStation = require('../../format/station')
```
Comment (derhuerst) [inline p/cmta/index.js]: Remove these 5 unused `require`s above.

**Comment by derhuerst** [comment]
Thanks for your contribution. This looks very good so far!

We should have made it more visible that [the `next` branch has a new profile format which makes it a lot easier to write profiles](https://github.com/public-transport/hafas-client/blob/856a751b513f6cb8d8a3afb4c2b8b6e22a397747/docs/writing-a-profile.md). I will add them to `next` though, so you won't have to do this.

File: p/cmta/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	return body
+}
+
+const defaultProducts = {
+  bus: true,
+  rapid: true,
+  rail: true
+}
+
+const formatBitmask = createFormatBitmask(modes)
```
Comment (derhuerst) [inline p/cmta/index.js]: This line ist duplicated at line `13`.

File: p/cmta/example.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+const cmtaProfile = require('.')
+
+const client = createClient(cmtaProfile)
+
+// Broadie Oaks to Domain
+client.journeys('000002370', '000005919', {results: 1, polylines: true})
+// client.departures('000002370', {duration: 1})
+// client.locations('Westgate', {results: 2})
+// client.location('000005534') // Downtown light rail station
+// client.nearby(30.266222, -97.746058, {distance: 60})
```
Comment (derhuerst) [inline p/cmta/example.js]: This should be an object. Check [the docs for `nearby()`](https://github.com/public-transport/hafas-client/blob/f3d8304db879ce54fef0e0a8daa1ef0a5e292c0b/docs/nearby.md).

**Comment by nickturskyi** [comment]
Thanks for your feedback @derhuerst !
One more thing - just realized that there are in fact tests for particular profiles.
Hold on, I'll add some tests for CMTA as well.

**Comment by derhuerst** [comment]
Please also add an entry to `readme.md`.

**Comment by derhuerst** [comment]
We've merged the tests PR (#57) into `next`.

@nickturskyi Do you want to write tests or should I do it?

**Comment by nickturskyi** [comment]
@derhuerst I'm going to add tests if that's not something you already started working on.
Thanks for merging!

**Comment by derhuerst** [comment]
Didn't start yet.

**Comment by derhuerst** [comment]
Started working on it. Adapted your code to the latest version of `hafas-client`.

Closing in favor of #79.

**Pull request closed without merging**